x-pack/plugins/security_solution/cypress/tasks/api_calls/spaces.ts

```diff
@@ -14,9 +14,17 @@ export const createSpace = (id: string) => {
       name: id,
     },
     headers: { 'kbn-xsrf': 'cypress-creds' },
+  }).then((response) => {
+    expect(response.status).equal(200);
   });
 };
 
 export const removeSpace = (id: string) => {
-  cy.request(`/api/spaces/space/${id}`);
```

Comment:
Did this ever work or was it just not used?

Reply:
This worked but a bit flaky, so re-writing it, hopefully it will be bit more stable.